.github/ISSUE_TEMPLATE/release-tracker.md

```diff
@@ -0,0 +1,30 @@
+---
+name: Release Tracker
+about: Release Tracker
+title: 'Release <x.y.z>'
+labels: release-tracker
+
+---
+
+<!--
+Fill in the actual release version.
+If the schedule release date is known, fill the actual date.
+-->
+
+**Scheduled release date: TBD**
+
+This is a tracker to aggregate issues that should be part of <x.y.z>. 
+Please add specific Issues or PRs prefixed with the "Depends on" keyword 
+(and remember that PRs need to be backported to the release-<x.y> branch).
+
+Make sure to follow the manual steps listed below before starting the release process.
+
+### Manual Steps 
+
+For detailed information regarding each step, please check: https://submariner.io/development/building-testing/ci-maintenance/
+- [ ] Custom GitHub Actions
+- [ ] GitHub Actions
+- [ ] Kubernetes Versions
+- [ ] Shipyard Base Image Software
+- [ ] Shipyard Linting Image Software
+
```

Comment:
Lets add an item to also update dependencies in submariner-addon after the release is done and 1 to start x+1.0.0-m0 after each rc0 releases.

Reply:
The addon is not this project’s concern and shouldn’t be tied to Submariner releases in Submariner documentation.

Reply:
@Jaanki I've added an item for m0